fission/src/ui/panels/configuring/assembly-config/interfaces/inputs/ConfigureSchemeInterface.tsx

```diff
@@ -93,6 +95,23 @@ const ConfigureSchemeInterface: React.FC<ConfigSchemeProps> = ({ selectedScheme
                         />
                     )
                 })}
+                {
+                    <Button
+                        value={"Add Joint Control"}
+                        onClick={() => {
+                            const jointIndex =
+                                Math.max(
+                                    0,
+                                    ...selectedScheme.inputs
+                                        .map(input => parseInt(input.inputName.replace("joint ", "")))
+                                        .filter(val => !isNaN(val))
+                                ) + 1
```

Comment:
We should probably extract the second argument (after the spread) into its own variable.